### ISSUE TITLE
[BUG]  2.4.1 (2.4.1+4899-09d279c6) Update triggers ends on dark page

### ISSUE THREAD

**Issue by Suenee** [bitfocus/companion#2232]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When I try to update any trigger, it ends on gray background of companion. It seems to me, that CP tries to reload page unsuccessfuly.

### Steps To Reproduce

1. go to Triggers
2. create new trigger
3. add feedbeck to atem8 to channel 7
4. AND add feedback to atem4 to channel 2
5. add feedback internal button background color on page 2 and bank 21
6. SAVE
7. Trigger window dismis
8. you see gray background of CP
9. CP web page is not working. you can only reload.
10. The same issue is when you try to update this trigger

### Expected Behavior

you have

Blackmagic ATEM Mini
Blackmagic ATEM Mini Extreme ISO

I think the issue is more about colors than BM switchers, because when I try to update any other trigger where is no color update, it works well.

### Environment (please complete the following information)

```markdown
- OS: Win7
- Browser: Chrome
- Companion Version: 2.4.1 (2.4.1+4899-09d279c6)
```

### Additional context

I have last version of CP. I made update today.

**Comment by jsnyder886** [comment]
I've noticed this behavior as well.

What I've found is that if in updating the condition of the trigger results
in the trigger firing, this behavior is observed.  If the updated
conditions do not result in the trigger immediately firing, the page
behaves normally as expected.

**Comment by Julusian** [comment]
this should be fixed in 2.4.2
